src/systems/systems/src/hydraulic/aerodynamic_model.rs

```diff
@@ -0,0 +1,669 @@
+use nalgebra::{Rotation3, Vector3};
```

Comment:
**question**: Should this module be a child of the `hydraulic` module? Is it only related to hydraulics or can these types be reused for other means?

Reply:
Generally speaking it could be used for other things, but for now don't really see any use case for other modules than hydraulics. 
Any evil ideas there? :p

Reply:
Nope. We could always move it later when we see a use for it elsewhere.